.github/workflows/windows.yml

```
@@ -0,0 +1,40 @@
+name: Build and Test on Windows
+on: [push]
+jobs:
+  windows:
+    defaults:
+      run:
+        shell: cmd
+    strategy:
+      matrix:
+          version: ["7.4", "8.0"]
+          arch: [x64]
+          ts: [ts]
+    runs-on: windows-latest
```

Maybe better:
```suggestion
    runs-on: windows-2019
```
`latest` is still an alias of `2019`, but that might change (`2022` is already available as beta)